static/js/components/search/HierarchicalRefinementRangeFilter.js

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import {HierarchicalRefinementFilter} from 'searchkit';
+import FinalGradeRangeFilter  from './FinalGradeRangeFilter';
+
+
+export default class HierarchicalRefinementRangeFilter extends HierarchicalRefinementFilter {
```

Comment:
This might be better named something like `CourseWithFinalGrade` in the name, this name sounds general-purpose but  that's not the case.